test/parallel/test-repl.js

```diff
@@ -134,6 +134,15 @@ function error_test() {
     // Uncaught error throws and prints out
     { client: client_unix, send: 'throw new Error(\'test error\');',
       expect: /^Error: test error/ },
+    // Throws primitive and prints out
+    { client: client_unix, send: 'throw null;',
+      expect: 'null' },
+    // Throws Error without stack and prints out
+    {
+      client: client_unix,
+      send: 'var e = new Error(\'test error\'); delete e.stack; throw e;',
+      expect: /^Error: test error/
+    },
```

Comment:
add this monstrosity:
`function MySillyError() {}; MySillyError.prototype = SyntaxError; var e = new MySillyError(); throw e`